### PR TITLE
Mobile: Simple chat attachments for photos

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

Here's a simple Chat uploader for Android and iOS -- it offers a choice between selecting an image from the Camera Roll and taking a new image with the camera. There's no way to change the image title so I've filed DESKTOP-2979 for adding that.

This PR plumbs onSelectAttachment() through to the main conversation view, since that's where it's needed on mobile.

### CODE REVIEW

File: shared/chat/conversation/container.js [hunk 1]
```diff
@@ -176,6 +176,10 @@ export default connect(
     onPostMessage: (selectedConversation, text) => dispatch(postMessage(selectedConversation, new HiddenString(text))),
     onRetryAttachment: (message: AttachmentMessage) => dispatch(retryAttachment(message)),
     onRetryMessage: (conversationIDKey: ConversationIDKey, outboxID: OutboxIDKey) => dispatch(retryMessage(conversationIDKey, outboxID)),
+    onSelectAttachment: (conversationIDKey: ConversationIDKey, input: AttachmentInput) => {
+      input.conversationIDKey = conversationIDKey
```
Comment (chromakode) [inline shared/chat/conversation/container.js]: Looks like `conversationIDKey` is set in `_openFilePicker`. Is this needed?

File: shared/react-native/ios/Keybase/Info.plist [hunk 2]
```diff
@@ -60,5 +60,7 @@
 	<false/>
 	<key>NSCameraUsageDescription</key>
 	<string>Scanning codes</string>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>Attaching images to Chat messages</string>
```
Comment (chromakode) [inline shared/react-native/ios/Keybase/Info.plist]: :sheep: Lowercase chat?

File: shared/chat/conversation/container.js (hunk 1 above)
Comment (cjb) [inline shared/chat/conversation/container.js]: Good call, not needed, thanks!

File: shared/react-native/ios/Keybase/Info.plist (hunk 2 above)
Comment (cjb) [inline shared/react-native/ios/Keybase/Info.plist]: :thumbsup:

File: shared/package.json [hunk 3]
```diff
@@ -75,6 +75,7 @@
     "react-list": "0.8.0",
     "react-native": "0.41.2",
     "react-native-camera": "0.6.0",
+    "react-native-image-picker": "^0.25.6",
```
Comment (chrisnojima) [inline shared/package.json]: lets remove the ^

**Comment by cjb** [comment]
Oops, thanks for catching the ^.

**Pull request merged**